crypto/src/hash/pedersen/mod.rs

```diff
@@ -0,0 +1,136 @@
+use super::traits::IsCryptoHash;
+use lambdaworks_math::{
+    cyclic_group::IsGroup,
+    elliptic_curve::{
+        self,
+        short_weierstrass::{
+            curves::bls12_381::{curve::BLS12381Curve, field_extension::BLS12381PrimeField},
+            element::ProjectivePoint,
+        },
+        traits::IsEllipticCurve,
+    },
+    field::{self, element::FieldElement, traits::IsField},
+    traits::ByteConversion,
+    unsigned_integer::element::U384,
+};
+
+type FE = field::element::FieldElement<BLS12381PrimeField>;
+type Point = ProjectivePoint<BLS12381Curve>;
+
+const WINDOW_SIZE: usize = 4;
+const NUM_WINDOWS: usize = 96;
```

Comment:
Because this value is 96, `HALF_INPUT_SIZE_BITS` is 192 which means `hash_two()` is only hashing the first element (since each element is 384 and both elements are being concatenated into a single buffer).